ui/src/app/cohort-search/attributes-page/attributes-page.component.html

```diff
@@ -1,78 +1,123 @@
-            <span *ngIf="units[node.get('subtype')]">{{units[node.get('subtype')]}}</span>
-          </div>
-          <div *ngIf="attrForm.value['operator'+i] === 'BETWEEN'" class="form-group and-text">
-            <div>and</div>
-          </div>
-          <div *ngIf="attrForm.value['operator'+i] === 'BETWEEN'" class="form-group range-container">
-            <input class="number-container" type="number" min="0" [name]="'valueB'+i" [(ngModel)]="attr.operands[1]" (ngModelChange)="inputChange()" required>
-            <span>{{units[node.get('subtype')]}}</span>
+    <div *ngIf="node.get('type') === criteriaTypes.MEAS">
+      <div class="attr-name-color">
+        {{displayName}}
+        <clr-icon shape="refresh" size="20" class="attr-refresh" (click)="refresh()"></clr-icon>
+      </div>
+      <div class="radio">
+        <input type="radio" name="labExists" [value]="true" [(ngModel)]="attrs.EXISTS" (change)="radioChange()">
+        <label>Any value (lab exists)</label>
+      </div>
+      <div *ngIf="!attrs.EXISTS && attrs.NUM.length" class="or-circle">OR</div>
+    </div>
+    <ng-container *ngIf="!attrs.EXISTS">
+      <div *ngFor="let attr of attrs.NUM; let i = index">
+        <div class="attr-name-color">{{dropdowns.labels[i]}}</div>
+          <div class="container">
+            <div class="dropdown-width">
+              <clr-dropdown>
+                <button type="button" class="btn btn-outline" clrDropdownTrigger>
+                  <span *ngIf="!attr.operator">Select Operator</span>
+                  <span *ngIf="attr.operator">{{dropdowns.selected[i]}}</span>
+                  <clr-icon shape="caret down"></clr-icon>
+                </button>
+                <clr-dropdown-menu clrPosition="bottom-left" *clrIfOpen>
+                  <button
+                  *ngFor="let option of options"
+                  [name]="'operator'+i"
+                  (click)="selectChange(i, option)"
+                  class="dropdown-item"
+                  type="button"
+                  clrDropdownItem>{{option.name}}</button>
+                </clr-dropdown-menu>
+              </clr-dropdown>
+              <input type="hidden" [name]="'operator'+i" [ngModel]="attr.operator" [required]="node.get('type') === criteriaTypes.PM">
+            </div>
+            <div *ngIf="attr.operator && attrForm.value['operator'+i] !== 'ANY'" class="form-group range-container">
+              <input class="number-container"
+                type="number"
+                [min]="attr.MIN"
+                [max]="attr.MAX"
+                [name]="'valueA'+i"
+                [(ngModel)]="attr.operands[0]"
+                (ngModelChange)="inputChange()"
+                required>
+              <span *ngIf="units[node.get('subtype')]">{{units[node.get('subtype')]}}</span>
+              <span *ngIf="node.get('type') === criteriaTypes.MEAS">
```

Comment:
same for these?

Reply:
Done. May need a follow-up PR to get these in other components since they are everywhere